### PR TITLE
APM-408: Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by Sourceless**
## Summary
* :robot: Add an azure pipeline to build, local test, and upload artifacts

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

**Comment by Sourceless** [comment]
@alexhawdon I think this one is ready to go, it's been stable for a little while now

File: proxies/live/apiproxy/policies/Quota.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Quota name="Quota">
+    <DisplayName>Quota</DisplayName>
+    <Allow count="300"/>
```
Comment (alexhawdon) [inline proxies/live/apiproxy/policies/Quota.xml]: Stick a TODO or FIXME in here so we come back to this? (And read it out of the API Product being used)

File: proxies/live/apiproxy/policies/SpikeArrest.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<SpikeArrest name="SpikeArrest">
+  <DisplayName>SpikeArrest</DisplayName>
+  <Rate>5ps</Rate>
```
Comment (alexhawdon) [inline proxies/live/apiproxy/policies/SpikeArrest.xml]: Same here maybe

File: proxies/sandbox/apiproxy/policies/AssignMessage.AddAsidHeader.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (alexhawdon) [inline proxies/sandbox/apiproxy/policies/AssignMessage.AddAsidHeader.xml]: Pretty sure all these policies are unused in the Sandbox. I'll make a ticket to refactor: APM-565

**Pull request merged**